lib/log_file.c

```diff
@@ -147,13 +153,9 @@ qb_log_file_close(int32_t t)
 int32_t
 qb_log_file_reopen(int32_t t, const char *filename)
 {
-	struct qb_log_target *target = qb_log_target_get(t);
 	int rc;
 
-	if (filename) {
-		(void)strlcpy(target->filename, filename, PATH_MAX);
-	}
-	rc = _do_file_reload(t);
+	rc = _do_file_reload(filename, t);
```

Comment:
Why not `return -errno; /* from fopen() */` instead of `return -1;` in `_do_file_reload()` so that you can then rename it to `int32_t qb_log_file_reopen()` and happily be one function down?